server/json-rpc/src/lib.rs

```diff
@@ -244,15 +254,25 @@ pub fn parse_response(response: Value) -> Result<(), jsonrpc_core::Error> {
 }
 
 fn subgraph_routes(name: &SubgraphName, http_port: u16, ws_port: u16) -> Value {
+    let http_base_url = EXTERNAL_HTTP_BASE_URL
+        .clone()
+        .unwrap_or_else(|| format!(":{}", http_port));
```

Comment:
Could we omit the port from the output if it is port 80? That would be nice.

Reply:
Yup, this allows for that! `http_base_url` is either something like `:8000`, in which case graph-cli fills in the protocol and host, or something like `http://example.com` or `http://example.com:8000`, in which case graph-cli will use the URL directly.